docs/container_setup/github_codespace/creating_codespace.md

```diff
@@ -19,3 +19,27 @@ GitHub Codespaces' button and then click the green 'Create Codespace' button.
 
 3. You will be taken to a VSCode editor within your browser.
 ![codespace](../../assets/rdev3.png)
+
+4. Rename your codespace for easier identification:
+
+By default, your codespace will have a randomly generated name. Renaming it to
```

Comment:
From this line onwards the indentation is incorrect. At least the first line of each paragraph and the first line of each bullet point should be indented four spaces so that it aligns to the right of the item number (`4.`). I recommend to indent all of the lines 4 spaces, to make it clearer in the markdown file that this content is part of listed item, rather than new content after the list.